File: TransCoder_tokenized_test_set_functions/cpp/SORTING_ARRAY_REVERSE_AROUND_MIDDLE.cpp
bool ifPossible ( int arr [ ], int n ) {
  int cp [ n ];
  copy ( arr, arr + n, cp );
  sort ( cp, cp + n );
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( ! ( arr [ i ] == cp [ i ] ) && ! ( arr [ n - 1 - i ] == cp [ i ] ) ) return false;
  }
  return true;
}

File: TransCoder_tokenized_test_set_functions/java/SORTING_ARRAY_REVERSE_AROUND_MIDDLE.java
static boolean ifPossible ( int arr [ ], int n ) {
  int copy [ ] = Arrays . copyOf ( arr, arr . length );
  Arrays . sort ( copy );
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( ! ( arr [ i ] == copy [ i ] ) && ! ( arr [ n - 1 - i ] == copy [ i ] ) ) return false;
  }
  return true;
}